### PR TITLE
Portfolio media query for desktop version pull request

### PR DESCRIPTION
**Pull request by codecaiine**
- I created a new branch for a desktop version.
- I used media query for a Desktop version.
- I replicated as much as possible Figma template 1 desktop version.
- I used media queries in my CSS file to make a desktop version of Figma template 1 .
- I used both Grid and Flexbox.
- I fixed linters errors for HTML and CSS.
- I updated my readme file with a description of the project.

 **This is my last code review. Feel free to contact me on slack if something's wrong [ Yannick-Noel ]** .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,13 @@ The project is inspired by the figma microverse template 1 (https://figma.com/)
...
+![screenshot](./image/sc-desktop1.PNG)
 
-
-- Screenshoot II
-![screenshot](./image/mob2.PNG)
-
-
-- Screenshoot III
-![screenshot](./image/mob3.PNG)
+![screenshot](./image/sc-desktop2.PNG)
```
Comment (carlos-ssh) [inline README.md]: - Add a description about the changes you made in the pull request description.

File: css/style.css [hunk 2]
```diff
@@ -12,6 +12,7 @@ html {
 body {
   font-weight: normal;
   color: #091e42;
+  font-size: 20px;
 }
```
Comment (carlos-ssh) [inline css/style.css:17]: - Kindly add @media screen and (min-width: 1080px) into your project. Read again the project requirements.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @codecaiine, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (codecaiine) [reply README.md]: Hello 
Thank you for your review and comments. 
Please can you explain this part?
Then I will understand better what I need to do

Comment (carlos-ssh) [reply README.md]: You are missing steps from the general requirements:
Screen Shot 2021-07-23 at 07 26 03.

Comment (codecaiine) [reply README.md]: Oh I got it thank you.

File: css/style.css (hunk 2 above)
Comment (codecaiine) [reply css/style.css:17]: You can find the media query I put in my style.css at line 448 . Please check the screenshot below

mediaquery

File: README.md (hunk 1 above)
Comment (codecaiine) [reply README.md]: I update it , thank you .

Descript

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**